src/components/Episodes/Episodes.jsx

```diff
@@ -9,7 +9,18 @@ import {
 
 export default function Episodes(){
 
-	const episodes = data.map(RENDER.episode);
+	const episodes = data.map( (obj, index) => {
+		//passing prev and next video src for video nav buttons
+		const prevTrailer = index !== 0 
+			? data[index-1].trailer
+			: {};
```

Comment:
Did we fix this on the other branch? Deffo shouldn't be duplicating work if so but if not then this should done in the render util